data/wanderer/wanderers start.txt

```diff
@@ -179,67 +183,388 @@ mission "Wanderers: Hai Diplomat"
-				`	"What if the humans want our own envoy to these aliens?"`
-					goto envoy
-			`	"Yes," she says. "If these aliens will provide a place for me, I mean to live with them permanently, an ambassador between their people and ours. I am old, and in need of a great novelty like this to make my life interesting once more." You note that the fur on her face, especially on the snout, has turned from brown to silvery gray - a sign of age in terrestrial creatures, and perhaps in the Hai as well.`
-			`	"Besides," she says, "after recent events they can hardly say no to me."`
+				`	"No, I'd rather wait for the Hai government to provide an ambassador."`
+					decline
+	on accept
+		log "Professor Andelena and her grad student agreed to act as interpreters for the aliens to the north. I need to meet them at the <planet> spaceport."
+
+
+
+mission "Wanderers: Intercepted"
+	invisible
+	source Allhome
+	to offer
+		has "Wanderers: Professor Available: done"
+		not "Wanderers: Diplomacy: active"
+		not "Wanderers: Diplomacy: done"
+		not "Wanderers: Sayari Available After Hai Reveal: offered"
+	to complete
+		never
+	on offer
+		fail "Wanderers: Meet Professor"
+		conversation
+			`As you enter the spaceport, a group of armed Hai in uniforms approach you. The uniforms match the one from the Hai that refused to help you on Hai-home.`
+			`	One of them addresses you. "Are you looking for Professor Andelena?"`
 			choice
-				`	"Let's go visit these aliens immediately!"`
-					accept
-				`	"What if the humans want our own envoy to these aliens?"`
-			label envoy
-			`	She says, "If they prove to be friendly, I will help you to find a way to communicate with them directly, so you need not use me as an intermediary. But our first step must be to speak with them." You help her aboard your ship, and get ready to return to the alien homeworld.`
+				`	"Yes."`
+					goto yes
+				`	"Is she in some kind of trouble?"`
+				`	"I've never heard that name in my life."`
+					decline
+			`	"Neither she nor you are in trouble. We just want you to help us clear up some matters."`
+				goto question
+			label yes
+			`	"There has been an incident and some miscommunications. We want you to help us clear up some matters."`
+			label question
+			choice
+				`	"Lead on."`
+				`	"No, I think I'll be on my way."`
+					decline
+			`	The armed Hai escort you to a secure room, where a worried Professor Andelena looks at her student, Eenap, who is in handcuffs. An stern-looking Hai woman in a similar uniform to the others seems to be in charge, and glares at Eenap before addressing you.`
+			`	"Your efforts to find an interpreter have stumbled upon a suspected Unfettered spy who tried to transmit details about you and your ship to the Unfettered." She glares again at Eenap, who parries with a defiant look. "Fortunately, we shut down the operation before it got too far. You are safe, for now, Captain <last>."`
```

Comment:
```suggestion
			`	"Your efforts to find an interpreter have stumbled upon a suspected Unfettered spy who tried to transmit details about you and your ship to the Unfettered." She glares again at Eenap, who parries with a defiant look. "Fortunately, we shut down the operation before it got too far. You are safe for now, Captain <last>."`
```

Reply:
The extra comma is to pause the reader at that point. It stresses the "for now."

Reply:
Resolvable than.